app/controllers/projects_controller.rb

```diff
@@ -64,18 +64,6 @@ def activity
     # No necessary code here
   end
 
-  def transfer
```

Comment:
Instead of removing this and moving the method into a new transfer controller can you move it back here. I did some digging and you are not using the `new` endpoint of the transfer controller you added and only the `create` as such I think it would be simpler to just keep this method and make it match the other method you had added.

Reply:
Sounds good, updated in 2703d62